docs/cli-tutorial.md

```diff
@@ -0,0 +1,738 @@
+planet data search PSScene -
+```
+
+The date input understands [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) and 
+[ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) formats. You can specify just the date,
+or include the time, to search for all Planetscope images acquired within 5 seconds at a time
+on July 1st 2021:
+
+```console
+planet data filter --date-range acquired gte 2021-07-01:06:20:10 --date-range acquired lt 2021-07-01:06:20:15 | \
+planet data search PSScene - 
+```
+
+#### Range Filter
+
+The range filter uses the same operators as the date filter, but works against any numerical property. The most useful
+of these tend to be ones about cloudy pixels. For example you can search for data with clear pixels greater than 90%:
+
+```console
+planet data filter --range clear_percent gt 90
+```
+
+#### String-In Filter
+
+For properties that are strings you can use the `string-in` filter. For example search for all planetscope imagery
+with PS2 instrument:
+
+```console
+planet data filter --string-in instrument PS2 | planet data search PSScene -
+```
+
+You can specify multiple strings to match, with a comma:
+
+```console
+planet data filter --string-in instrument PS2,PSB.SD | planet data search PSScene -
+```
+
+Another example is to select all data in a single strip:
+
+```console
+planet data filter --string-in strip_id 5743640 | planet data search PSScene -
+```
+
+Note that in all these commands we are piping the results into the search. If you don't include the pipe then you'll
+get the filter output, which can be interested to inspect to see exactly what is sent to the server.
+
+#### Filter by asset
+
+You can limit your search to only data with a particular asset, for example search just for 8-band analytic assets:
+
+```console
+planet data filter --asset ortho_analytic_8b_sr | planet data search PSScene -
+```
+
+Or 8-band assets that also have a UDM.
+
+```console
+planet data filter --asset ortho_analytic_8b_sr --asset udm2 | planet data search PSScene -
+```
+
+You can find the list of available assets in each Item Type Page, like 
+[available assets](https://developers.planet.com/docs/data/psscene/#available-asset-types) for PSScene. You can see
+[a table of all Item Types](https://developers.planet.com/docs/data/psscene/#available-asset-types), which links to 
+the page for each with their list of asset types.
+
+Note that the asset filter doesn't perform any validation, so if your searches aren't returning anything check to make
+sure you got the asset right, and it's valid for the item-types you're searching.
+
+#### Permission Filter
+
+The 'permission filter' is set to true by default, since most people want to search only for data they have access to
+and are able to download. But if you'd like to just get search Planet's catalog and get a sense of what is out there
+you can set the permission filter to false:
```

Comment:
how does this work? They have to have an api key to do anything, right? And if they have an api key then what is the limitation on their download access?

Reply:
Maybe I have this wrong? Yes, they need an API key, but I was thinking if 'permission filter' was set to true then they'd only see data api results in the AOI that they have purchased access to. And if they wanted to search outside then they'd need to set it to false?

Though now I'm questioning - how does the permission filter work? What is the effect of setting it to true or false? 

This is one of those times it'd be nice to easily test the non-planeteer experience.

Reply:
hmm yeah I agree it would be helpful to be able to test different scenarios. @sarasafavi is there any way for devrel to get a set of tester API keys?